.github/workflows/changelog-and-release.yml

```diff
@@ -93,9 +93,9 @@ jobs:
           then
             changes=$(xmlstarlet fo -R "$(find . -name addon.xml.in)" | xmlstarlet sel -t -v 'string(/addon/extension/news)' | awk -v RS= 'NR==1')
           fi
-          echo "changes=<<EOF" >> $GITHUB_OUTPUT
+          echo "changes<<EOFn" >> $GITHUB_OUTPUT
           echo "$changes" >> $GITHUB_OUTPUT
```

Comment:
Should this be `echo "changes=$changes" >> $GITHUB_OUTPUT`?

Reply:
Or does the first line essentially do that assingnment?

Reply:
Ok, that is exactly the case. I sent an update PR that introduces best practice for EOF and makes the same changes for release.yml: https://github.com/xbmc/inputstream.adaptive/pull/1487

Reply:
> Should this be `echo "changes=$changes" >> $GITHUB_OUTPUT`?

this is right but only when you have a string, without multilines